web/profiles/joinup/joinup.behat.inc

```diff
@@ -2006,4 +2006,55 @@ class JoinupSubContext extends DrupalSubContextBase implements DrupalSubContextI
     \PHPUnit_Framework_Assert::assertEquals($text, $link->getText());
   }
 
+  /**
+   * Asserts that an tag with the given text exists in the given region.
+   *
+   * This completes the assertNotRegionElementText of Markup context.
+   *
+   * @see \Drupal\DrupalExtension\Context\MarkupContext::assertNotRegionElementText()
+   *
+   * @Then I( should) see :text in the :tag element in the :region( region)
```

Comment:
I think the step could be improved a bit, as actually it checks that any :tag element has the wanted text, and there might be more than one :tag in the region.
`@Then I( should) see a :tag (HTML )element with the text :text in the :region( region)`